libbeat/docs/communitydashboards.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[community-beat-dashboards]]
+== Community Beat dashboards
+
+The open source community has been hard at work developing new Beats. You can check
```

Comment:
Hmmm...not sure this intro makes sense, even if you change it to say :"The open source community has been hard at work developing new Beats dashboards." Maybe we could say something simple like, "To help you visualize data in Kibana, here are some dashboards contributed by the Beats development community." 

Since the list is empty right now, it sounds a bit odd to say that people have been hard at work. :-) 

I understand that you want to have the link active, but it's too bad that we can't seed this list with a couple of dashboards so contributors will have an example. Or maybe that's possible?